### PR TITLE
Make collection report consistent

### PR DESCRIPTION
**Pull request by Miauwkeru**
(DIS-1935)

### CODE REVIEW

File: acquire/acquire.py [hunk 1]
```diff
@@ -866,17 +866,17 @@ class AV(Module):
         ("glob", "sysvol/ProgramData/Emsisoft/Reports/scan*.txt"),
         # F-Secure
         ("dir", "sysvol/ProgramData/F-Secure/Log"),
-        ("dir", "sysvol/Users*/AppData/Local/F-Secure/Log"),
+        ("glob", "sysvol/Users*/AppData/Local/F-Secure/Log"),
```
Comment (Schamper) [inline acquire/acquire.py:869]: Why not make this use from_user_home too?

File: acquire/acquire.py [hunk 2]
```diff
@@ -866,17 +866,17 @@ class AV(Module):
...
         ("dir", "sysvol/ProgramData/HitmanPro/Logs"),
         ("dir", "sysvol/ProgramData/HitmanPro.Alert/Logs"),
         ("file", "sysvol/ProgramData/HitmanPro.Alert/excalibur.db"),
-        ("glob", "sysvol/ProgramData/HitmanPro/Quarantine"),
+        ("dir", "sysvol/ProgramData/HitmanPro/Quarantine"),
         # Malwarebytes
         ("glob", "sysvol/ProgramData/Malwarebytes/Malwarebytes Anti-Malware/Logs/mbam-log-*.xml"),
         ("glob", "sysvol/ProgramData/Malwarebytes/MBAMService/logs/mbamservice.log*"),
-        ("dir", "sysvol/Users*/AppData/Roaming/Malwarebytes/Malwarebytes Anti-Malware/Logs"),
+        ("glob", "sysvol/Users*/AppData/Roaming/Malwarebytes/Malwarebytes Anti-Malware/Logs"),
```
Comment (Schamper) [inline acquire/acquire.py:879]: Why not make this use from_user_home too?

**Review by Schamper — CHANGES_REQUESTED** [review]

File: tests/test_collector.py [hunk 3]
```diff
@@ -247,15 +263,84 @@ def test_collector_collect_path_no_file_type(mock_target, mock_collector) -> Non
...
+)
+def test_collector_report_succeeded(
+    tmp_path: Path,
+    mock_collector: Collector,
+    function_name: str,
+    collection_point: str,
+    expected_results: int,
+    create_paths: list[str],
+):
+    target = create_target_with_files(tmp_path, create_paths)
```
Comment (Poeloe) [inline tests/test_collector.py:341]: Why not using the mock_target within the conftest?

**Review by Poeloe — CHANGES_REQUESTED** [review]

File: tests/test_collector.py [hunk 4]
```diff
@@ -247,15 +263,78 @@ def test_collector_collect_path_no_file_type(mock_target, mock_collector) -> Non
...
+            1,
+            ["collect/this/file"],
+        ),
+        (
+            "glob",
+            "/collect/*/file",
+            0,
+            [],
+        ),
+        ("file", "collect/this/file", 1, ["collect/this/file"]),
```
Comment (Poeloe) [inline tests/test_collector.py:321]: For consistency, also put al the values of this tuple on separate lines

File: acquire/acquire.py [hunk 5]
```diff
@@ -866,17 +866,17 @@ class AV(Module):
...
-        ("glob", "sysvol/ProgramData/HitmanPro/Quarantine"),
+        ("dir", "sysvol/ProgramData/HitmanPro/Quarantine"),
         # Malwarebytes
         ("glob", "sysvol/ProgramData/Malwarebytes/Malwarebytes Anti-Malware/Logs/mbam-log-*.xml"),
         ("glob", "sysvol/ProgramData/Malwarebytes/MBAMService/logs/mbamservice.log*"),
-        ("dir", "sysvol/Users*/AppData/Roaming/Malwarebytes/Malwarebytes Anti-Malware/Logs"),
+        ("dir", "AppData/Roaming/Malwarebytes/Malwarebytes Anti-Malware/Logs", from_user_home),
         ("dir", "sysvol/ProgramData/Malwarebytes/MBAMService/ScanResults"),
         # McAfee
         ("dir", "sysvol/Users/All Users/Application Data/McAfee/DesktopProtection"),
```
Comment (pyrco) [inline acquire/acquire.py:882]: This one can also use `from_user_home`

**Review by pyrco — CHANGES_REQUESTED** [review]
I'd rename the `test_glob.py`  to something more descriptive and more in line with how we want to call our unittest, like `test_acquire_modules.py`.

**Review by pyrco — APPROVED** [review]

**Pull request merged**